tests/unit/Responses/GetRecordingsTextTracksResponseTest.php

```diff
@@ -40,7 +40,9 @@ public function testGetRecordingTextTracksResponseTypes()
 
         $secondTracks = $this->tracks->getTracks()[1];
 
-        $this->assertEachGetterValueIsString($secondTracks, [
+        $this->assertEachGetterValueIsString(
+            $secondTracks,
+            [
```

Comment:
I don't like this style.
@sualko @FelixJacobi Any ideas how to keep the old formatting?

Reply:
It is caused by this rule https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/function_notation/method_argument_space.rst

In version 2 the default was to ingnore not full multiline argument lists. In version 3 the PSR-12 standard is used, which requires all arguments to be in a seperate line, see: https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments

@sualko @FelixJacobi What behavior would you prefer. After seeing it in PSR-12 I changed my mind and would prefer the suggested change.

Reply:
It is caused by this rule https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/function_notation/method_argument_space.rst

In version 2 the default was to ingnore not full multiline argument lists. In version 3 the PSR-12 standard is used, which requires all arguments to be in a seperate line, see: https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments

@sualko @FelixJacobi What behavior would you prefer. After seeing it in PSR-12 I changed my mind and would prefer the suggested change.

Reply:
>  After seeing it in PSR-12 I changed my mind and would prefer the suggested change.

Would be fine for me.